loadbalancing_101.rst

```diff
@@ -4,23 +4,29 @@ Load Balancing
 Why do we use load balancers?
 =============================
 
-* A fast, HTTP/TCP based load balancer can distribute between slower application
-  servers, and can keep each application server running optimally
-* They keep application servers busy by buffering responses and serving them to
-  slow clients (or keepalives). We want app servers to do real work, not waste
-  time waiting on the network.
-* Load balancers provide a mechanism to verify the health of backend servers to
-  ensure traffic is only routed to backends available to service the request.
-  These health checks can be simplistic ICMP pings to ensure host availability
-  to advanced HTTP (layer 7) health checks that use HTTP response codes or
-  content matching.
-* Can provide a layer of abstraction such that end user endpoints remain
-  consistent (i.e URLs) as backend application infrastructure expands
-  (or contracts)
-* Can be either software based (aka reverse proxies) or hardware based (physical
-  devices optimized for network throughput and ISO layer 2,3 and 4 routing).
-* Can be tuned by itself for max conns?
-* HTTP path based routing, 1 domain to multiple heterogeneous server pools.
+Scalability
+-----------
+
+The most basic scalability need is to allow one address handle a load that's too much for one physical server.
+A secondary scalability factor is offloading or reducing underlying tasks, such as SSL termination and TCP re-use.
+Smart load balancers that are able to poll and react to client load can seemlessly distribute work among heterogenous machines.
```

Comment:
What does "TCP re-use" mean here? Socket reuse?

> able to poll

Not sure what that means (well, I think I know what you're trying to say, but others may not).